game/scripts/npc/abilities/ancient_apparition_ice_blast.txt

```diff
@@ -63,7 +63,7 @@
       "05"
       {
         "var_type"                                        "FIELD_FLOAT"
-        "frostbite_duration"                              "10 11 12 13 14"
+        "frostbite_duration"                              "10 11 12 14 16" //OAA
```

Comment:
Revert this change. AA's ultimate lasts long enough for it to be effective. Can you instead reduce the mana cost on chilling touch to something reasonable at OAA levels.

Reply:
this is a needed change to AA. 

As baumi once said "if u want to buff a hero, buff it's strengths"

xd